packages/fluent/scss/coloreditor/index.scss

```diff
@@ -21,8 +21,8 @@ $_kendo-module-meta: (
 @include module.register( $_kendo-module-meta... );
 
 // Expose
-@mixin coloreditor-styles() {
-    @include module.render( "coloreditor" ) {
+@mixin color-editor-styles() {
```

Comment:
Another thing I am curious about - we are streamlining the names of the components here e.g. coloreditor -> coloreditor, colorgradient -> color-gradient, however do we also want the mixin names to also match?

Currently there is a difference between Fluent - `@mixin color-editor-styles()` and the rest of the themes - `@mixin color-editor--styles()`

Reply:
It would be great if the names are the same across all themes, however we can leave this as a separate tasks for a future improvement.

If we want the Fluent theme to import the Default theme and customize it via variables (like the other themes do), then we would need to have the same names across all themes.